docs/Using_marketplace.md

```diff
@@ -0,0 +1,74 @@
+# How to use the DeFi Data Marketplace
+This marketplace allows search, shop and download DeFi data using a simple web application or a command-line interface. This guide will explain how to use the web application to download DeFi Data, to use the command line interface follow [this guide](https://defi.mumbai.nevermined.rocks/cli)
+
+## Searching datasets
+In order to find the dataset to download, the search bar can be used, searching for a specific protocol or using the filters to find the type of event that you are interested in. This search bar can be found at the top of the page. 
+
+![banner](./assets/searchbar.png)
+
+To filter the data, several filters can be applied. In order to find the event type that we're looking for, the category filter can be used. With it, we can specify which specific type of event we want to search for. These events are the different types of user interactions that are currently supported in the marketplace. 
+
+Also, other advanced filters can be used, to only return datasets that cames from a specific blockchain, publication date or based on the price. 
+
+## Buying datasets
+Once we know which datasets we want to download, we need to add them to the shopping cart. There are several ways to do a show. 
```

Comment:
is this expression correct? `There are several ways to do a show` I would just write this:
`There are several ways to do:`